### ISSUE TITLE
RSX unnecessarily initializes cache when stopping after Emu.Ready

### ISSUE THREAD

**Issue by Megamouse** [RPCS3/rpcs3#10848]
1. Deselect "Start games after boot"
2. Boot a game
3. Instead of pressing Play, press Stop
4. -> RSX initializes cache including some overlays
5. RSX gets aborted

This happens because - while the Emu is only ready - the thread waits for us to start emulation without honoring that the emulation might have been stopped instead.
So step 4 above can be skipped in order to prevent any unwanted behaviour.

**Comment by digant73** [comment]
@Megamouse although I can imagine the reply, is that still an issue after 4 years? just in case you forgot an update

**Comment by elad335** [comment]
Yes but I am against "solving" it. It's not safe to skip initialization of RSX. The benefit of addressing this issue is minimal.
